tests/Jenkins/dirac_ci.sh

```diff
@@ -119,18 +119,14 @@ function installSite(){
 
   installOptions="$DEBUG "
 
-  # If DIRACOSVER is not defined, use LcgBundle
   if [ "$DIRACOSVER" ]; then
-    if [ "$DIRACOSVER" == True ]; then
-      echo "Installing with DIRACOS"
-      installOptions+="--dirac-os "
-    else
-      echo "Installing with DIRACOS $DIRACOSVER"
-      installOptions+="--dirac-os --dirac-os-version=$DIRACOSVER "
-    fi
-  else
-    echo "Installing using externals + lcgBundle"
-    installOptions+="-t fullserver "
+    installOptions+="--dirac-os --dirac-os-version=$DIRACOSVER "
+  fi
+
+  if [ "$DIRACOS_TARBALL_PATH" ]; then
+    {
+      echo "DIRACOS = $DIRACOS_TARBALL_PATH"
+    } >> "$SERVERINSTALLDIR/dirac-ci-install.cfg"
```

Comment:
There is already a "install.cfg" file created at runtime, why not use that one?

Reply:
It seemed cleaner to me as this way the code is the same for both the client and server install. (I'll probably unify them as part of extending this to the LHCbDIRAC CI.)